### DISCUSSION TITLE
Question: HTTPS supported?

### DISCUSSION THREAD

**Discussion by Grandma-Betty** [djdd87/SynoAI#129] [Q&A] [answered]
I have connected SSS via HTTPS to my cameras. Now I'm wondering if I can set up SynoAI for HTTPS enabled cameras instead of using HTTP. Will this work? If so, is there something special to be aware of when configuring the docker container?
Thanks!

**Comment by djdd87** [comment, marked as answer]
The cameras themselves are irrelevant as far as SynoAI is concerned as it just goes via SurveillanceStation. So as to whether the camera is being accessed over HTTP/HTTPS/Magic/etc, SynoAI is unaware and doesn't care.

**Reply by Grandma-Betty** [reply]
Thanks mate!
